src/config.zig

```diff
@@ -168,7 +168,7 @@ const ConfigCluster = struct {
     /// It is used to assert that all cluster members share the same config.
     pub fn checksum(comptime config: ConfigCluster) u128 {
         @setEvalBranchQuota(10_000);
-        var hasher = std.crypto.hash.Blake3.init(.{});
+        var hasher = std.crypto.hash.blake2.Blake2s256.init(.{});
```

Comment:
Is Blake3 == Blake2s256? If it is, might make sense to add a comment here. Also, maybe `TODO: switch to vsr.checksum once that works at comptime`? (I tihnk I have an issue tracking that,but comment won't hurt)

Reply:
Good point on the comment. It was changed due to Blake3 in stdlib having a compile error internally.